client/my-sites/theme/main.jsx

```diff
@@ -101,6 +102,30 @@ import ThemeStyleVariations from './theme-style-variations';
 
 import './style.scss';
 
+const LivePreviewButton = ( { siteSlug, themeId, themeType } ) => {
+	let themePath = '';
+	switch ( themeType ) {
+		case 'free':
+			themePath = 'pub/';
+			break;
+		case 'premium':
+			themePath = 'premium/';
+			break;
+	}
+
+	if ( themePath === '' ) {
+		return null;
+	}
```

Comment:
Am I correct in understanding that we can only get free themes and premium themes on Simple sites since it respects `wp-content/themes/*`?
Or do we have the path to get marketplace or woo commerce theme? (BTP gets a theme with the string specified in the URL query `theme_preview`. https://github.com/WordPress/gutenberg/blob/trunk/lib/compat/wordpress-6.3/theme-previews.php#L20-L21)

Reply:
All themes on simple sites are located under `wp-content/themes/` folder, so we're not able to preview 3rd party themes via BTP as wpcom doesn't install those themes.

Reply:
Thank you for confirming! I'll comment on the CfT post about this.